.github/workflows/release.yml

```diff
@@ -29,6 +29,4 @@ jobs:
         run: >
           git config --global user.email $${{ secrets.EMAIL }} &&
           git config --global user.name ${{ secrets.NAME }} &&
-          git add . &&
-          git commit -m "New Version" &&
```

Comment:
This shouldn't be necessary to get the GH and NPM releases, as `auto` does the job. Here I am just aligning with `addon-kit` but happy to revert this if you do prefer to have release commits.

It could be a good idea in that case to add a `[skip-ci]` or `[skip-release]` tag on the commit message to avoid any future issues, as CD tools tend to look for those messages to avoid triggering loops.